.rubocop

```diff
@@ -0,0 +1,5 @@
+--parallel
+--display-cop-names
+--extra-details
+--display-style-guide
```

Comment:
More verbose error messages (Display cop names is default, but I've included it here for completeness)

Reply:
yep the more detail the better, will save some googling